advocate_europe/templates/formtools/wizard/wizard_form.html

```diff
@@ -0,0 +1,50 @@
+{% extends "base.html" %}
+{% load widget_tweaks i18n %}
+
+{% block content %}
+    <div class="custom-form container-medium">
+        <form class="custom-form-wizard" method="post" enctype="multipart/form-data">
+        <div class="row">
+            <div class="col-md-6">
+                <h1>{% trans 'create an idea' %} </h1>
+            </div>
+            <div class="col-md-6 form-section_title">
+                <h1>{{ form.section_name }}</h1>
+            </div>
+        </div>
+
+
+        {% csrf_token %}
+        {{ wizard.form.media }}
+        {{ wizard.management_form }}
+        {% if wizard.form.forms %}
```

Comment:
I am not sure. I simply overwrote this template here: https://github.com/django/django-formtools/blob/master/formtools/templates/formtools/wizard/wizard_form.html